### ISSUE TITLE
'fomosto build --force' doesn't work? 

### ISSUE THREAD

**Issue by axelwang** [pyrocko/pyrocko#327]
Hey,

  I am working with a Green function store initialized using QSSP.

  I read on your Fomosto tutorial that when I change parameters in the `config` or `extra/qssp` files, I should use `fomosto ttt --force` and `fomosto build --force` to rebuild the traces. However, while `fomosto ttt --force`  always works fine, `fomosto build --force` always doesn't work. When I change a parameter from either `config` or `extra/qssp`, I get a very long error by trying to rebuild using `fomosto build --force`. The error begins with: 
`fomosto: error: ===== begin qssp input =====
# autogenerated QSSP input by qssp.py
#
# This is the input file of FORTRAN77 program "qssp2010" for calculating
# synthetic seismograms of a self-gravitating, spherically symmetric,
# isotropic and viscoelastic earth.
`
Basically it's a long list of the inputs to QSSP. Then, the error message ends with
`Error in qpwvint: t/f sampling inconsistent with Green functions!
                 ntcut             dt  nfcut             df
 Current input:     75      5.00000000     64      0.00156250
     Data base:     95      5.00000000     64      0.00156250
===== end qssp output =====
===== begin qssp error =====
===== end qssp error =====
the string 'error' appeared in qssp output
qssp has been invoked as "fomosto_qssp2010" `

  I am not sure if this is because I didn't use the commands correctly or if 'fomosto build --force' actually doesn't work? 

  Thanks,
  Axel

**Comment by emolch** [comment]
For QSSP, you have to delete the `qssp_green` subdirectory additionally.

This is not documented anywhere. I am short on time atm. Would you be willing to fix this documentation issue?

**Comment by axelwang** [comment]
Hey Emolch,
  Thank you for the explanation! It works now! I will add it when I can!

  Best,
  Axel
